.circleci/config.yml

```diff
@@ -18,6 +18,9 @@ workflows:
       - test:
           requires:
             - build
+      - test-node-v12:
+          requires:
+            - build-node-v12
```

Comment:
Do we need a new build step? Can't we just run the existing build in a new test step?

Reply:
@webmaster128 Very much WIP, I'll ping you when this is ready for review.